app/views/layouts/_messages.html.erb

```diff
@@ -1,12 +1,14 @@
 <% unless flash.empty? %>
-  <% flash.each do |name, msg| %>
-    <% if msg.is_a?(String) %>
-      <% if name == 'notice' then name = 'info' end %>
-      <div class="usa-alert usa-alert__paragraph usa-alert--<%= name %>" >
-        <div class="usa-alert__body">
-          <p class="usa-alert__text"><%= msg %></p>
+  <div class="margin-bottom-2">
+    <% flash.each do |name, msg| %>
+      <% if msg.is_a?(String) %>
+        <% if name == 'notice' then name = 'info' end %>
+        <div class="usa-alert usa-alert__paragraph usa-alert--<%= name %>" >
+          <div class="usa-alert__body">
+            <p class="usa-alert__text"><%= sanitize(msg) %></p>
```

Comment:
Do we need to explicitly allow `<a>` tags in `sanitize`? Or does this go away if we suffix the `_html` key correctly? (sometimes serializing things to a flash loses that data...)

Reply:
We'll still need `sanitize`, I think, but it's working as expected with no other modifications (tested both locally and in my personal sandbox).